pouchDB/pouchdb.d.ts

```diff
@@ -0,0 +1,273 @@
+// Type definitions for Pouch 0.1
+// Project: http://pouchdb.com
+// Definitions by: Viktor Zhakhalov <https://github.com/zhakhalov/>
+// Definitions: https://github.com/borisyankov/DefinitelyTyped
+// Based on https://github.com/MrBigDog2U/DefinitelyTyped/blob/master/pouchDB/
+
+
+declare var PouchDB: pouchdb.PouchDB;
+
+declare module 'pouchdb' {
+  export = PouchDB;
+}
+
+declare module pouchdb {
+
+  export interface IPromise<T> {
```

Comment:
http://www.typescriptlang.org/Handbook#writing-dts-files

> Naming Conventions
In general, do not prefix interfaces with I (e.g. IColor). Because the concept of an interface in TypeScript is much more broad than in C# or Java, the IFoo naming convention is not broadly useful.